docker/funsize-update-generator/scripts/funsize.py

```diff
@@ -25,6 +25,9 @@
     "http://download.mozilla.org/",
 ]
 
+DEFAULT_FILENAME_TEMPLATE = "{appName}-{branch}-{version}-{platform}-" \
+                            "{locale}-{from_buildid}-{to_buildid}.partial.mar"
```

Comment:
That's OK. releasetasks set their own template, this is the default (backward compatible).